templates/jaasai/index.html

```diff
@@ -203,10 +203,10 @@ <h2>Why use Juju?</h2>
     </div>
     <div class="row">
       <div class="col-6 u-align-text--center">
-        <img src="/static/img/home/why-use-juju-diagram-left.svg" />
+        <img class="lazy" data-src="/static/img/home/why-use-juju-diagram-left.svg" />
```

Comment:
This class means that if the user has JS disabled they won't get any of these images.